releaseNotes/AzureAppConfigurationProviderPython.md

```diff
@@ -2,6 +2,34 @@
 
 [Source code][source_code] | [Package (Pypi)][package] | [Samples][samples]
 
+
+## 1.0.0 - March 9, 2023
+
+### Breaking Changes
+
+* Renamed `load_provider` method to `load`
+* Added `AzureAppConfigurationKeyVaultOptions` to take in a `client_configs` Mapping of endpoints to client `kwargs` instead of taking in the whole client.
+* Removed `AzureAppConfigurationKeyVaultOptions` secret_clients, client_configs should be used instead.
+* Made key_filter and label_filter kwargs for Setting Selector
+* Renamed trimmed_key_prefixes to trim_prefixes
+
+## 1.0.0b2 - February 15, 2023
+
+### New Features
+
+* Added Async Support
+* Full support of Mapping API
```

Comment:
Worth an explanation what's "Mapping API".

Reply:
Added a bit of one.